scripts/operator/raise_pr_to_community_operators_from_our_fork.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
+
+import os
+from github import Github
+from get_last_published_operator_version import getLastOperatorVersion
+import pr_resources
+
+GITHUB_AUTOMATIC_OPERATOR_PRS_ACCESS_TOKEN = os.environ[
+    'GITHUB_AUTOMATIC_OPERATOR_PRS_ACCESS_TOKEN']
+
+github = Github(GITHUB_AUTOMATIC_OPERATOR_PRS_ACCESS_TOKEN)
+
+
+def get_latest_operator_branch():
+    repo = github.get_repo('snyk/community-operators')
+    branches = repo.get_branches()
+    operator_branches = []
+
+    for branch in branches:
+        if 'snyk-operator' in branch.name:
+            operator_branches.append(branch.name)
+
+    return operator_branches[-1]
```

Comment:
FYI - a shorter option (haven't checked the syntax :) )
```suggestion
    return [branch.name for branch in branches if 'snyk-operator' in branch.name][-1]
```